go.mod

```diff
@@ -12,7 +12,7 @@ require (
 	github.com/go-ozzo/ozzo-validation v3.6.0+incompatible
 	github.com/go-sql-driver/mysql v1.4.1 // indirect
 	github.com/golang/protobuf v1.3.2
-	github.com/google/uuid v1.1.1 // indirect
+	github.com/google/uuid v1.1.1
```

Comment:
this is suspicous. I think we use the satori package everywhere else

Reply:
That is true, from Hub

![image](https://user-images.githubusercontent.com/891889/75968104-2b8a1e00-5ecd-11ea-96b3-92abbe5b0b19.png)

Reply:
This PR only reverts changes does not make them. Answering you question: not everywhere, there is an HTTP middleware that uses a wrong one. Feel free to open a PR to fix that https://github.com/contiamo/go-base/blob/2180135922a7fd1ff8c7502c3cb5c6b9e28da6ee/pkg/http/middlewares/path.go#L9

Reply:
:+1: Just noticed that. It often happens that the auto-importer picks the wrong package, thats very likely here. I'm going to have a look over lunch

Reply:
But that is weird, this PR even reverts that pkg from satori to google 🤔 
![image](https://user-images.githubusercontent.com/891889/75968494-c71b8e80-5ecd-11ea-8831-4d479bd7802a.png)

Reply:
@LucasRoesler because I found and fixed it during my refactoring, so now I'm reverting everything back.

Reply:
Again, if anyone feels like it, please open a PR fixing this, but it does break anything to use the Google library there.